src/components/tl/ucp/tl_ucp.c

```diff
@@ -140,6 +140,10 @@ ucc_config_field_t ucc_tl_ucp_lib_config_table[] = {
      ucc_offsetof(ucc_tl_ucp_lib_config_t, allgather_kn_radix),
      UCC_CONFIG_TYPE_UINT},
 
+     {"ALLGATHER_USE_CUDA", "1", "Flag weather to use mc cuda copy",
```

Comment:
Should it be "Flag whether to use network loopback for cuda copy"? I think "mc cuda" is just saying it's GPU memory

Reply:
If so, should the flag be "ALLGATHER_USE_LOOPBACK"?